src/lib/hyrise.cpp

```diff
@@ -1,8 +1,16 @@
 #include "hyrise.hpp"
 
+#include <jemalloc/jemalloc.h>
+
 namespace opossum {
 
 Hyrise::Hyrise() {
+  // Work around for a performance regression in jemalloc:
+  // https://github.com/jemalloc/jemalloc/issues/1677
+  // Other jemalloc tuning options can be added here as found necessary.
+  auto huge_threshold = 0;
```

Comment:
Did we play around with this value? In the referenced issue, the jemalloc developer states that 40M may be an appropriate value for the benchmark we showed them.

Reply:
Nope. For now, we don't really care about anyone else on the same machine, so giving back memory is not really an issue. I don't really want to open that box in addition to all the other open issues.